src/pages/Profile.jsx

```diff
@@ -80,6 +84,13 @@ const Profile = () => {
     >
       <Box sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center', gap: '20px' }}>
         <Typography sx={{ fontWeight: 'bold', fontSize: '18px' }}>Profile Information</Typography>
+        <Typography>
+          {client ?? (
```

Comment:
So I just tested out the latest version of Development and am getting an app-breaking error from this component, saying that `client is not defined`.

It seems like `client` in this line is supposed to be `contact` instead.

I see a couple of other times `client` is used in this component but those don't break functionality.

Reply:
Good catch. Think we could open a bug report for it.